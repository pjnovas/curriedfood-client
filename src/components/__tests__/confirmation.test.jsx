import { noop } from 'lodash';
import Confirmation from '../confirmation';

const props = {
  onDismiss: noop,
  onConfirm: noop,
  title: 'Title',
  children: 'content',
  okText: 'OK!'
};

takeSnapshots(
  {
    default: {},
    visible: {
      ...props,
      visible: true
    },
    loading: {
      ...props,
      visible: true,
      loading: true
    }
  },
  Confirmation
);
