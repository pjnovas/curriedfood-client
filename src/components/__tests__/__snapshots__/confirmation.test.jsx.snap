// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default 1`] = `
<withTheme(Dialog)
  dismissable={true}
  style={
    Object {
      "backgroundColor": "#263238",
    }
  }
>
  <withTheme(Dialog.Title)
    style={
      Object {
        "color": "#ffffff",
        "textAlign": "center",
      }
    }
  />
  <Dialog.Content />
  <Dialog.Actions>
    <withTheme(Button)>
      Cancelar
    </withTheme(Button)>
    <withTheme(Button)
      dark={true}
      mode="contained"
      onPress={[Function]}
      style={
        Object {
          "marginHorizontal": 10,
        }
      }
    />
  </Dialog.Actions>
</withTheme(Dialog)>
`;

exports[`loading 1`] = `
<withTheme(Dialog)
  dismissable={false}
  onDismiss={[Function]}
  style={
    Object {
      "backgroundColor": "#263238",
    }
  }
  visible={true}
>
  <withTheme(Dialog.Title)
    style={
      Object {
        "color": "#ffffff",
        "textAlign": "center",
      }
    }
  >
    Title
  </withTheme(Dialog.Title)>
  <Dialog.Content>
    content
  </Dialog.Content>
  <Dialog.Actions>
    <withTheme(Button)
      disabled={true}
      onPress={[Function]}
    >
      Cancelar
    </withTheme(Button)>
    <withTheme(Button)
      dark={true}
      loading={true}
      mode="contained"
      onPress={[Function]}
      style={
        Object {
          "marginHorizontal": 10,
        }
      }
    >
      OK!
    </withTheme(Button)>
  </Dialog.Actions>
</withTheme(Dialog)>
`;

exports[`visible 1`] = `
<withTheme(Dialog)
  dismissable={true}
  onDismiss={[Function]}
  style={
    Object {
      "backgroundColor": "#263238",
    }
  }
  visible={true}
>
  <withTheme(Dialog.Title)
    style={
      Object {
        "color": "#ffffff",
        "textAlign": "center",
      }
    }
  >
    Title
  </withTheme(Dialog.Title)>
  <Dialog.Content>
    content
  </Dialog.Content>
  <Dialog.Actions>
    <withTheme(Button)
      onPress={[Function]}
    >
      Cancelar
    </withTheme(Button)>
    <withTheme(Button)
      dark={true}
      mode="contained"
      onPress={[Function]}
      style={
        Object {
          "marginHorizontal": 10,
        }
      }
    >
      OK!
    </withTheme(Button)>
  </Dialog.Actions>
</withTheme(Dialog)>
`;
